go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/google/renameio v0.1.0
 	github.com/imdario/mergo v0.3.12
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v0.0.0-20220110171111-997c316db5e1
+	github.com/openshift/api v0.0.0-20220203140920-bfe251c51d2d
```

Comment:
Actually I think https://github.com/openshift/machine-config-operator/pull/2949/files uses a later api (20220209124712-b632c5fc10c0) here so will probably want to rebase and pick that up instead of this one (if my eyes aren't deceiving  me)

Reply:
Indeed, We will just rebase that in.